pipelines/template-build/template-build.yaml

```diff
@@ -87,9 +87,9 @@ spec:
           workspace: git-auth
     - name: prefetch-dependencies
       when:
-      - input: $(params.hermetic)
-        operator: in
-        values: ["true"]
+      - input: $(params.prefetch-input)
+        operator: notin
+        values: [""]
```

Comment:
I'm not sure if this is true of `cachi2`, but in `cachito` there was a default value if `""` was used.

Reply:
I was poking around [here](https://github.com/containerbuildsystem/cachi2/blob/main/cachi2/interface/cli.py#L162) and it seems like some value is needed. Not 100% sure, but even there was a default, I think we could reasonably say that users of this task need to be explicit and not rely on the default.

Reply:
Agree